models/xchange/ethereum/xchange_ethereum_schema.yml

```diff
@@ -0,0 +1,97 @@
+version: 2
+
+models:
+  - name: xchange_ethereum_trades
+    meta:
+      blockchain: ethereum
+      sector: dex
+      project: xchange
+      contributors: mike-x7f
+    config:
+      tags: ['ethereum','xchange','x7finance','trades','dex']
+    description: >
+        xchange contract trades on Ethereum
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - block_date
+            - blockchain
+            - project
+            - version
+            - tx_hash
+            - evt_index
+            - trace_address
+      - check_dex_seed:
+          blockchain: ethereum
+          project: xchange
+          version: 1
```

Comment:
In order for `check_dex_seed` to work, you need to add seed records in the following file:
https://github.com/duneanalytics/spellbook/blob/main/seeds/dex/trades/dex_trades_seed.csv